inst/qml/NonParametricSurvivalAnalysis.qml

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick			2.8
+import QtQuick.Layouts	1.3
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+
+Form
+{
+	VariablesForm
+	{
+		AvailableVariablesList
+		{
+			name: "allVariablesList"
+		}
+
+		AssignedVariablesList
+		{
+			name:				"timeToEvent"
+			title:				qsTr("Time to Event")
+			suggestedColumns:	["scale"]
+			singleVariable:		true
+		}
+
+		AssignedVariablesList
+		{
+			id:					eventStatusId
+			name:				"eventStatus"
+			title:				qsTr("Event Status")
+			suggestedColumns:	["nominal"]
+			singleVariable:		true
+		}
+
+		DropDown
+		{
+			name:				"eventIndicator"
+			label:				qsTr("Event Indicator")
+			source:				[{name: "eventStatus", use: "levels"}]
+			onCountChanged:		currentIndex = 1
+		}
+
+		AssignedVariablesList
+		{
+			name:			 	"factors"
+			title:			 	qsTr("Factors")
+			allowedColumns:		["ordinal", "nominal", "nominalText"]
+		}
+	}
+
+	Group
+	{
+
+		CheckBox
+		{
+			name:	"tests"
+			label:	qsTr("Tests")
```

Comment:
Is it necessary to nest the individual tests under the `tests` checkbox? Why not just use a `Group{}`? This feels a bit convoluted to me.

Reply:
fixed